src/DynamoRevit/DynamoRevit.csproj

```diff
@@ -47,7 +47,12 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="DynamoApplications">
-      <HintPath>$(DYNAMOAPI)\DynamoApplications.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\..\Program Files\Dynamo\Dynamo Core\1.2\DynamoApplications.dll</HintPath>
```

Comment:
This change is not required, we still need to use $(DYNAMOAPI) variable to reference any dynamo specific dll